package.json

```diff
@@ -82,7 +83,7 @@
     "eslint-plugin-markdown": "^2.2.1",
     "eslint-plugin-prefer-let": "^3.0.1",
     "express": "^4.17.1",
-    "jest": "^27.5.1",
+    "jest": "^28.1.0",
```

Comment:
This is required in order to 

```ts
import { getTestServer } from "@google-cloud/functions-framework/testing";
```
https://github.com/penx/remix/blob/29be5a6c90e9d1a02db06d6e9f306947ef89d452/packages/remix-google-cloud-functions/__tests__/server-test.ts#L10-L11

Reply:
```ts
import { getTestServer } from "@google-cloud/functions-framework/testing";
```

This import makes use of [package exports](https://nodejs.org/api/packages.html#exports):

https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/759aac18ad297363c4651efcf19ab2bf8fa9625a/package.json#L13

Full support for this [was added in Jest 28](https://jestjs.io/blog/2022/04/25/jest-28#packagejson-exports).

Without Jest 27,  server-test.ts will fail saying it cannot find `@google-cloud/functions-framework/testing`.

Reply:
Let's do this in a separate PR instead, so we keep this PR focussed.

Reply:
Let's do this in a separate PR instead, so we keep this PR focussed.

I think the Jest 28 change is also causing some tests to fail, so we could solve them in isolation in the new PR.

Reply:
@penx Now that https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/461 is merged, do we still need `jest` v28?